mkdocs_click/_extension.py

```diff
@@ -21,10 +21,11 @@ def replace_command_docs(**options: Any) -> Iterator[str]:
     command = options["command"]
     prog_name = options.get("prog_name", command)
     depth = int(options.get("depth", 0))
+    styles = {"option": options.get("option-style", "plain")}
```

Comment:
Should we simply do…?

```suggestion
    style = options.get("style", "plain")
```

If there's a need for "different kinds of styles" in the future we can still go back and refactor things, but for now there's a single style value that we need to manipulate.